File: app/src/main/java/com/example/letmebeyourchef/Listeners/IstruzioniListener.java
package com.example.letmebeyourchef.Listeners;

import com.example.letmebeyourchef.Models.ResponseFromApiIstruzioni;

import java.util.List;

public interface IstruzioniListener {
    void didFetch(List<ResponseFromApiIstruzioni> response, String message);
    void didError(String message);
}

File: app/src/main/java/com/example/letmebeyourchef/Listeners/IstruzioniListener.kt
package com.example.letmebeyourchef.Listeners

import com.example.letmebeyourchef.Models.ResponseFromApiIstruzioni

open interface IstruzioniListener {
    fun didFetch(response: List<ResponseFromApiIstruzioni>, message: String?)
    fun didError(message: String?)
}